### PR TITLE
IPFS bump [HOTFIX]

### PR DESCRIPTION
**Pull request by simonovic86**
- bump `IPFS` and related dependencies
- fix lint errors

### CODE REVIEW

File: packages/ceramic-cli/src/ceramic-daemon.ts [hunk 1]
```diff
@@ -34,8 +33,8 @@ export interface CreateOpts {
 }
 
 interface HttpLog {
-  request: object;
-  response?: object;
+  request: Record<string, unknown>;
```
Comment (PaulLeCam) [inline packages/ceramic-cli/src/ceramic-daemon.ts:36]: Could use the Request/Response types from Express maybe if it's matching?

File: packages/ceramic-cli/src/ceramic-cli-utils.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import path from "path"
 import { randomBytes } from '@stablelib/random'
 import * as u8a from 'uint8arrays'
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const fs = require('fs').promises
```
Comment (PaulLeCam) [inline packages/ceramic-cli/src/ceramic-cli-utils.ts:7]: `import { promises as fs } from 'fs'`?

**Review by PaulLeCam — APPROVED** [review]
Great being able to use the main IPFS package again!

File: packages/ceramic-cli/src/ceramic-cli-utils.ts (hunk 2 above)
Comment (simonovic86) [reply packages/ceramic-cli/src/ceramic-cli-utils.ts]: 👍

File: packages/ceramic-cli/src/ceramic-daemon.ts (hunk 1 above)
Comment (simonovic86) [reply packages/ceramic-cli/src/ceramic-daemon.ts:36]: nope, those are different objects

**Pull request merged**